### PR TITLE
feat: add sample code for delete document schema

### PR DESCRIPTION
**Pull request by paulinawins**
## Description

Fixes #issue

Note: Before submitting a pull request, please open an issue for discussion if you are not associated with Google.

## Checklist
- [x] I have followed guidelines from [CONTRIBUTING.MD](https://github.com/GoogleCloudPlatform/nodejs-docs-samples/blob/main/CONTRIBUTING.md) and [Samples Style Guide](https://googlecloudplatform.github.io/samples-style-guide/)
- [x] **Tests** pass:   `npm test` (see [Testing](https://github.com/GoogleCloudPlatform/nodejs-docs-samples/blob/main/CONTRIBUTING.md#run-the-tests-for-a-single-sample))
- [x] **Lint** pass:   `npm run lint` (see [Style](https://github.com/GoogleCloudPlatform/nodejs-docs-samples/blob/main/CONTRIBUTING.md#style))
- [ ] These samples need a new **API enabled** in testing projects to pass (let us know which ones)
- [ ] These samples need a new/updated **env vars** in testing projects set to pass (let us know which ones)
- [x] This pull request is from a branch created directly off of `GoogleCloudPlatform/nodejs-docs-samples`. Not a fork.
- [ ] This sample adds a new sample directory, and I updated the [CODEOWNERS file](https://github.com/GoogleCloudPlatform/nodejs-docs-samples/blob/main/CODEOWNERS) with the codeowners for this sample
- [ ] This sample adds a new sample directory, and I created [GitHub Actions workflow](https://github.com/GoogleCloudPlatform/nodejs-docs-samples/blob/main/CONTRIBUTING.md#adding-new-samples) for this sample
- [ ] This sample adds a new **Product API**, and I updated the [Blunderbuss issue/PR auto-assigner](https://github.com/GoogleCloudPlatform/nodejs-docs-samples/blob/main/.github/blunderbuss.yml) with the codeowners for this sample
- [x] Please **merge** this PR for me once it is approved

### CODE REVIEW

**Comment by holtskinner** [comment]
Add a test for this function. Similar to what was done in https://github.com/GoogleCloudPlatform/nodejs-docs-samples/pull/3292

File: document-warehouse/delete-document-schema.js [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+  // Delete Document Schema
+  async function deleteDocumentSchema() {
+    // Initialize request argument(s)
+    const request = {};
+
+    // The full resource name of the location, e.g.:
+    // projects/{project_number}/locations/{location}/documentSchemas/{document_schema_id}
+    request.name = `projects/${projectId}/locations/${location}/documentSchemas/${documentSchemaId}`;
+
+    console.log(request);
```
Comment (telpirion) [inline document-warehouse/delete-document-schema.js:48]: Remove or print a more meaningful message to stdout.

File: document-warehouse/delete-document-schema.js [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+    // Make Request
+    const response = serviceClient.deleteDocumentSchema(request);
```
Comment (telpirion) [inline document-warehouse/delete-document-schema.js:51]: Use `await` here.

File: document-warehouse/delete-document-schema.js [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+
+    // Print out response
+    response.then(
```
Comment (telpirion) [inline document-warehouse/delete-document-schema.js:54]: We typically use `await` to force a method call to become blocking. Remove the `then()` call here.

File: document-warehouse/delete-document-schema.js [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+      result =>
+        console.log(
+          `Success!\nDocument Schema Deleted: \n${JSON.stringify(result)}`
+        ),
+      error => console.log(`Failed!\n${error}`)
```
Comment (telpirion) [inline document-warehouse/delete-document-schema.js:59]: Remove "Success!" and "Failure!" characters from messages.

File: document-warehouse/delete-document-schema.js [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+    );
+  }
+
+  // [END contentwarehouse_delete_document_schema]
+  await deleteDocumentSchema();
+}
+
+main(...process.argv.slice(2)).catch(err => {
```
Comment (telpirion) [inline document-warehouse/delete-document-schema.js:67]: Remove CLI setup:
https://googlecloudplatform.github.io/samples-style-guide/#no-cli

Instead, export the main method as `deleteDocumentSchema`.

**Review by telpirion — CHANGES_REQUESTED** [review]
This sample requires an integration test:
https://googlecloudplatform.github.io/samples-style-guide/#testing

There are some stylistic differences in this sample from the norm. I suggest that you look at  the following examples:

+ [Sample](https://github.com/GoogleCloudPlatform/nodejs-docs-samples/blob/main/speech/adaptationInlinePhraseSet.v2.js)
+ [Test](https://github.com/GoogleCloudPlatform/nodejs-docs-samples/blob/main/speech/system-test/adaptationInlinePhraseSet.v2.test.js)

**Review by paulinawins — COMMENTED** [review]
Done. Please review and approve when you have the time. 
@telpirion

**Pull request closed without merging**